thread/unix/thrdsup.c

```diff
@@ -72,6 +72,8 @@ intptr_t sem_destroy_zos(j9sem_t s);
 intptr_t sem_wait_zos(j9sem_t s);
 intptr_t sem_trywait_zos(j9sem_t s);
 intptr_t sem_post_zos(j9sem_t s);
+int setenv (const char *, const char *, int) __THROW;
+char *   getenv (const char *) __THROW;
```

Comment:
formatting doesn't match the [coding standard](https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md). which error does this change resolve?
```suggestion
int setenv(const char *, const char *, int) __THROW;
char *getenv(const char *) __THROW;
```

Reply:
Undid these changes, seems I had an older snapshot of `omr` where these declarations were completely missing and I had added badly formatted versions of already existing declarations.